src/main/java/io/kestra/plugin/googleworkspace/calendar/EventCreatedTrigger.java

```diff
@@ -0,0 +1,506 @@
+            code = """
+                id: google_calendar_meeting_trigger
+                namespace: company.team
+
+                tasks:
+                  - id: notify_meeting
+                    type: io.kestra.plugin.notifications.slack.SlackIncomingWebhook
+                    url: "{{ secret('SLACK_WEBHOOK') }}"
+                    payload: |
+                      {
+                        "text": "New meeting scheduled: {{ trigger.summary }} on {{ trigger.start.dateTime }}"
+                      }
+
+                triggers:
+                  - id: watch_meetings
+                    type: io.kestra.plugin.googleworkspace.calendar.EventCreatedTrigger
+                    serviceAccount: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
+                    calendarId: "team-calendar@company.com"
+                    q: "meeting"
+                    interval: PT10M
+                """
+        ),
+        @Example(
+            title = "Monitor multiple calendars with organizer filter",
+            full = true,
+            code = """
+                id: google_calendar_multiple_trigger
+                namespace: company.team
+
+                tasks:
+                  - id: log_event
+                    type: io.kestra.plugin.core.log.Log
+                    message: "Event by {{ trigger.organizer.email }}: {{ trigger.summary }}"
+
+                triggers:
+                  - id: watch_multiple_calendars
+                    type: io.kestra.plugin.googleworkspace.calendar.EventCreatedTrigger
+                    serviceAccount: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
+                    calendarIds:
+                      - primary
+                      - "team-calendar@company.com"
+                      - "project-calendar@company.com"
+                    organizerEmail: "manager@company.com"
+                    interval: PT15M
+                """
+        )
+    }
+)
+public class EventCreatedTrigger extends AbstractCalendarTrigger implements PollingTriggerInterface, TriggerOutput<EventCreatedTrigger.EventMetadata> {
+
+    @Schema(
+        title = "The Google Cloud service account key",
+        description = "Service account JSON key with access to Google Calendar API"
+    )
+    @NotNull
+    protected Property<String> serviceAccount;
+
+    @Schema(
+        title = "Calendar ID to monitor",
+        description = "Calendar ID (e.g., 'primary' or a calendar email). Use this OR calendarIds, not both."
+    )
+    protected Property<String> calendarId;
+
+    @Schema(
+        title = "Multiple Calendar IDs to monitor",
+        description = "List of calendar IDs to monitor. Use this OR calendarId, not both."
+    )
+    protected Property<List<String>> calendarIds;
+
+    @Schema(
+        title = "Free-text search filter",
+        description = "Only events matching this search term will trigger. Searches across title, description, and location."
+    )
+    protected Property<String> q;
```

Comment:
Let's name things meaningfully 🙏 : 

```suggestion
    protected Property<String> searchQuery;
```

Reply:
Don't forget to update the examples accordingly